canonical-schema.json

```diff
@@ -1,158 +1,200 @@
-          , "properties" :
-                { "uuid"        : { "$ref": "#/definitions/uuid" }
-                , "reimplements": { "$ref": "#/definitions/uuid" }
-                , "description" : { "$ref": "#/definitions/description" }
-                , "scenarios"   : { "$ref": "#/definitions/scenarios"   }
-                , "comments"    : { "$ref": "#/definitions/comments"    }
-                , "property"    : { "$ref": "#/definitions/property"    }
-                , "input"       : { "$ref": "#/definitions/input"       }
-                , "expected"    : { "$ref": "#/definitions/expected"    }
-                }
-          , "additionalProperties": false
-          },
-
-      "labeledTestGroup":
-          { "description": "A group of tests with a description"
-          , "type"       : "object"
-          , "required"   : ["description", "cases"]
-          , "properties" :
-                { "description": { "$ref": "#/definitions/description" }
-                , "scenarios"  : { "$ref": "#/definitions/scenarios"   }
-                , "comments"   : { "$ref": "#/definitions/comments"    }
-                , "cases"      : { "$ref": "#/definitions/testGroup"   }
-                }
-          , "additionalProperties": false
-          },
-
-      "description":
-          { "description": "A short, clear, one-line description"
-          , "type"       : "string"
-          },
-
-      "uuid":
-          { "description": "A universally unique identifier in the v4 format"
-          , "type"       : "string"
-          , "pattern"    : "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$"
-          },
-
-      "scenario":
-          { "description": "An identifier for a specific scenario (kebab-case)"
-          , "type": "string"
-          , "enum": ["big-integer", "floating-point", "library-test", "unicode"]
-          },
-
-      "scenarios":
-          { "description": "An array of scenarios that can be used to include/exclude test cases"
-          , "type"       : "array"
-          , "items"      : { "$ref": "#/definitions/scenario" }
-          , "minItems"   : 0
-          },
-
-      "property":
-          { "description": "A letters-only, lowerCamelCase property name"
-          , "type"       : "string"
-          , "pattern"    : "^[a-z]+([A-Z][a-z]+)*[A-Z]?$"
-          },
-
-      "input":
-          { "description": "The inputs to a test case"
-          , "type"       : "object"
-          },
-
-      "expected":
-          { "description": "The expected return value of a test case"
-          , "properties":
-               { "error": { "$ref": "#/definitions/error" }
-               }
-          , "dependencies":
-               { "error": { "maxProperties": 1 }
-               }
-          },
-
-      "error":
-          { "description": "A message describing an error condition"
-          , "type"       : "string"
-          }
-
+  "comments": [
```

Comment:
This (and the last few PRs) all changed the schema formatting. Is that intended?

Reply:
That's what prettier is doing. Schema isn't ignored.